tests/web3js/build_evm_state_test.js

```diff
@@ -41,7 +41,10 @@ it('should handle a large number of EVM interactions', async () => {
     let senderBalance = await web3.eth.getBalance(conf.eoa.address)
     assert.equal(senderBalance, 1999999999937000000n)
 
-    let transferAmounts = ['0.01', '0.03', '0.05']
```

Comment:
When the transfer amount `0.05` was randomly selected 3 times, it would cause the test to fail, as the EOA only has `0.15` ether.